src/platform/forms-system/src/js/components/FieldTemplate.jsx

```diff
@@ -46,16 +47,31 @@ export default function FieldTemplate(props) {
   let errorSpanId;
   let errorSpan;
   let errorClass;
+  let errorStyle = 'none';
   if (hasErrors) {
     errorClass = isDateField ? 'input-error-date' : 'usa-input-error';
     errorSpanId = `${id}-error-message`;
+    errorStyle = 'block';
     errorSpan = (
       <span className="usa-input-error-message" role="alert" id={errorSpanId}>
         <span className="sr-only">Error</span> {rawErrors[0]}
       </span>
     );
   }
 
+  if (environment.isStaging()) {
+    errorSpan = (
+      <span
+        style={{ display: errorStyle }}
+        className="usa-input-error-message"
+        role="alert"
+        id={errorSpanId}
+      >
+        <span className="sr-only">Error</span> {rawErrors[0]}
```

Comment:
@U-DON I've been seeing some errors on staging (see [this Slack thread](https://dsva.slack.com/archives/CBU0KDSB1/p1570802221045500)), possibly from this line. I think it might need to be updated to `{hasErrors && rawErrors[0]}`